react/features/invite/components/InfoDialogButton.web.js

```diff
@@ -19,26 +25,197 @@ const DEFAULT_BUTTON_CONFIGURATION = {
+     * instance is to be initialized.
+     */
+    constructor(props) {
+        super(props);
+
+        /**
+         * The timeout to automatically show the {@code InfoDialog} if it has
+         * not been shown yet in a lonely call.
+         *
+         * @type {timeoutID}
+         */
+        this._autoShowTimeout = null;
+
+        this.state = {
+            /**
+             * Whether or not the dialog has displayed at least once.
+             */
+            hasToggled: false
+        };
+
+        // Bind event handlers so they are only bound once for every instance.
+        this._onDialogClose = this._onDialogClose.bind(this);
+        this._onDialogToggle = this._onDialogToggle.bind(this);
+    }
+
+    /**
+     * Set a timeout to automatically show the {@code InfoDialog}.
+     *
+     * @inheritdoc
+     */
+    componentDidMount() {
+        this._autoShowTimeout = setTimeout(() => {
+            this._maybeAutoShowDialog();
+        }, INFO_DIALOG_AUTO_SHOW_TIME);
+    }
+
+    /**
+     * Update the state when the {@code InfoDialog} has been seen for the first
+     * time.
+     *
+     * @inheritdoc
+     */
+    componentWillReceiveProps(nextProps) {
+        if (!this.state.hasToggled && nextProps._showDialog) {
+            this.setState({ hasToggled: true });
+        }
+    }
+
+    /**
+     * Clear the timeout to automatically show the {@code InfoDialog}.
+     *
+     * @inheritdoc
+     */
+    componentWillUnmount() {
+        clearTimeout(this._autoShowTimeout);
+    }
+
+    /**
      * Implements React's {@link Component#render()}.
      *
      * @inheritdoc
      * @returns {ReactElement}
      */
     render() {
+        const { _showDialog, _visible, tooltipPosition } = this.props;
+        const buttonConfiguration = {
+            ...DEFAULT_BUTTON_CONFIGURATION,
+            classNames: [
+                ...DEFAULT_BUTTON_CONFIGURATION.classNames,
+                _showDialog ? 'toggled button-active' : ''
+            ]
+        };
+
         return (
-            <ToolbarButtonWithDialog
-                { ...this.props }
-                button = { DEFAULT_BUTTON_CONFIGURATION }
-                content = { InfoDialog } />
+            <InlineDialog
+                content = { <InfoDialog onClose = { this._onDialogClose } /> }
+                isOpen = { _visible && _showDialog }
+                onClose = { this._onDialogClose }
+                position = { TOOLTIP_TO_POPUP_POSITION[tooltipPosition] }>
+                <ToolbarButton
+                    button = { buttonConfiguration }
+                    onClick = { this._onDialogToggle }
+                    tooltipPosition = { tooltipPosition } />
+            </InlineDialog>
         );
     }
+
+    /**
+     * Callback invoked after a timeout to trigger display of the
+     * {@code InfoDialog} if certain conditions are met.
+     *
+     * @private
+     * @returns {void}
+     */
+    _maybeAutoShowDialog() {
```

Comment:
Ok, now I see what you meant. However, we want to always show the info dialog at the beginning of the call, not only when we're in a lonely call.